Umee-Bots/High-Utilization-Ratio/src/agent.ts

```diff
@@ -0,0 +1,150 @@
+import BigNumber from "bignumber.js";
+import {
+  BlockEvent,
+  ethers,
+  Finding,
+  getEthersProvider,
+  HandleBlock,
+  HandleTransaction,
+  Initialize,
+  TransactionEvent,
+} from "forta-agent";
+import CONFIG from "./agent.config";
+import { GET_RESERVES_LIST_ABI, GET_RESERVE_DATA_ABI, RESERVE_INITIALIZED_ABI } from "./constants";
+import { MulticallProvider } from "./multicall";
+import {
+  AgentConfig,
+  arrayChunks,
+  createAbsoluteThresholdFinding,
+  createPercentageThresholdFinding,
+  createReserveData,
+  ReserveBalances,
+  ReserveData,
+  usageRatio,
+} from "./utils";
+
+BigNumber.set({ DECIMAL_PLACES: 18 });
+
+const reserveData: ReserveData[] = [];
+
+// testing
+export const resetReserveData = () => (reserveData.length = 0);
+export const getReserveData = () => reserveData;
+export const setReserveData = (newReserveData: ReserveData[]) => {
+  resetReserveData();
+  newReserveData.forEach((el) => reserveData.push(el));
+};
+
+let multicallProvider: MulticallProvider;
+
+export const provideInitialize = (provider: ethers.providers.Provider, config: AgentConfig): Initialize => {
+  return async () => {
+    const LendingPool = new ethers.Contract(
```

Comment:
why not camelCase?

Reply:
This is technically a style rule break, but I do think it looks better in this case to use PascalCase since that's the way it's used in solidity and ethers basically mimicks that.

Reply:
I prefer to follow the same style everywhere